### ISSUE TITLE
Vagrant ports for sensu-enterprise-server are off

### ISSUE THREAD

**Issue by ghoneycutt** [sensu/sensu-puppet#735]
Currently one cannot test the API when using sensu-enterprise-server from the host. I believe that the redirected ports should be different in the Vagrantfile.

**Comment by jeffmccune** [comment]
The vagrant port does seem to be off by one (4568 instead of 4567) which may be a red herring though.  I'm looking into why there's nothing listening on port 4567 or 4568 inside the underlying VM, which is likely the root cause of the issue.

**Comment by jeffmccune** [comment]
Also observed this, which may be directly related to why the API isn't bound to the TCP port:

```
[vagrant@sensu-server ~]$ sudo sensu-enterprise start
OpenJDK 64-Bit Server VM warning: INFO: os::commit_memory(0x000000077ad30000, 2060255232, 0) failed; error='Cannot allocate memory' (errno=12)
#
# There is insufficient memory for the Java Runtime Environment to continue.
# Native memory allocation (malloc) failed to allocate 2060255232 bytes for committing reserved memory.
# An error report file with more information is saved as:
# /tmp/jvm-19823/hs_error.log
```

**Comment by ghoneycutt** [comment]
Fix released in v2.22.0
